src/components/AbstractSelectableComponent.js

```diff
@@ -124,21 +124,33 @@ export default class AbstractSelectableComponent extends React.Component {
   }
 
   render() {
+    const { tool } = this.state;
     const toolProps = {
       setActiveTool: (toolUpdate) => { this.setState({ tool: toolUpdate }); },
       /* eslint-disable react/destructuring-assignment */
       isActiveTool: toolCheck => (toolCheck === this.state.tool),
       /* esline-enable */
     };
 
-    const deckProps = {
+    let deckProps = {
       views: [new OrthographicView({ id: 'ortho' })], // id is a fix for https://github.com/uber/deck.gl/issues/3259
       layers: this.renderLayers().concat(this.renderSelectionLayers()),
       initialViewState: this.getInitialViewState(),
       onViewStateChange: this.onViewStateChange,
-      controller: true,
-      getCursor: interactionState => (interactionState.isDragging ? 'grabbing' : 'default'),
     };
+    if (tool) {
+      deckProps = {
+        controller: { dragPan: false },
+        getCursor: () => 'crosshair',
+        ...deckProps,
+      };
+    } else {
+      deckProps = {
+        controller: true,
+        getCursor: interactionState => (interactionState.isDragging ? 'grabbing' : 'default'),
+        ...deckProps,
+      };
+    }
```

Comment:
I think you could keep the const with something like:
```javascript
let deckProps = {
  views: [new OrthographicView({ id: 'ortho' })], // id is a fix for https://github.com/uber/deck.gl/issues/3259
  layers: this.renderLayers().concat(this.renderSelectionLayers()),
  initialViewState: this.getInitialViewState(),
  onViewStateChange: this.onViewStateChange,
  ...(tool ?
    { controller: { dragPan: false },
        getCursor: () => 'crosshair' }
    : { controller: true,
        getCursor: interactionState => (interactionState.isDragging ? 'grabbing' : 'default') }
  )
}
```
... but after typing it out, I'm pretty sure yours is less confusing. 😄